coalib/output/Logging.py

```diff
@@ -15,18 +15,24 @@ def configure_logging():
     logging.config.dictConfig({
         'version': 1,
         'handlers': {
-            'colored': {
+            'console-logger': {
                 'class': 'logging.StreamHandler',
-                'formatter': 'colored',
+                'formatter': 'color',
```

Comment:
Just for the sake of it. Color and no-color, instead of colored and non-colored. 
This setting doesn't have any effect outside this function. So it doesn't matter much.